BlueprintUI/Sources/Internal/ElementPath.swift

```diff
@@ -1,31 +1,28 @@
 /// Represents a path into an element hierarchy.
 /// Used for disambiguation during diff operations.
-struct ElementPath: Hashable {
+struct ElementPath: Hashable, CustomDebugStringConvertible {
```

Comment:
The hop to the inner instance type is no longer faster; in fact its slower than just using a native struct.